src/components/assets/Erc20Currency.vue

```diff
@@ -151,12 +151,17 @@ export default defineComponent({
       return getErc20Explorer({ currentNetworkIdx: currentNetworkIdx.value, tokenAddress });
     });
 
-    // Memo: Remove after runtime upgrading in astar network to enable EVM withdrawal
     const isDisabledXcmButton = computed(() => {
       const chainInfo = store.getters['general/chainInfo'];
       const chain = chainInfo ? chainInfo.chain : '';
       const currentNetworkIdx = getProviderIndex(chain);
-      return currentNetworkIdx === endpointKey.ASTAR && token.symbol !== 'DOT';
+
+      // Memo: Remove after runtime upgrading in shinde
+      const isMovr = token.symbol === 'MOVR';
```

Comment:
constants like 'MOVR' should be in separate file?

Reply:
How about like this? https://github.com/AstarNetwork/astar-apps/pull/471/commits/48f0575750ef2a737cee6b0657b90a2c5c1d79cc#diff-7abaf44bac320f8fd60f01ae1e14b5dc7a609393e9a6f2da2cecba2b078d99b1R160
However, this variable will be removed after runtime upgrading.